connect/models/schemas.py

```diff
@@ -315,11 +317,11 @@ class ParamSchema(BaseSchema):
     value = fields.Str()
     value_error = fields.Str()
     value_choice = fields.Str(many=True)
-
     title = fields.Str()
     scope = fields.Str()
     constraints = fields.Nested(ConstraintsSchema)
     value_choices = fields.Nested(ValueChoiceSchema, many=True)
+    structured_value = fields.Dict()
```

Comment:
from models/param.py schema it is `List[str]` or `Optional[str]`. So here is the 3d type for this parameter in this PR

Reply:
fixed in /models/param.py